<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramelessWindowsClass</class>
 <widget class="QWidget" name="FramelessWindowsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FramelessWindows</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FramelessWindowsClass {
background-color: lightblue;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#bt_close,#bt_maxmize,#bt_minmize {
background-color: lightyellow;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
font: bold 14px;
min-width: 5em;
padding: 6px;
min-height:1em;
}
QPushButton#bt_close:pressed,#bt_maxmize:pressed,#bt_minmize:pressed {
background-color: rgb(224, 0, 0);
border-style: inset;
color:yellow;
}
QPushButton#bt_close:hover,#bt_maxmize:hover,#bt_minmize:hover{
color: lightblue;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>275</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_minmize">
        <property name="text">
         <string>MinMize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_maxmize">
        <property name="text">
         <string>MaxMize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
min-height:2em;
max-width:10em;
background-color:lightyellow;
border:none;
}
QComboBox::drop-down{
subcontrol-origin: padding;
subcontrol-position: top right;
width: 15px;
padding-right:6px;
border:none;
}
QComboBox::down-arrow{
image:url(:/FramelessWindows/images/down.svg);
width: 15px;
height:15px;
}
QComboBox QAbstractItemView {
border: 2px solid darkgray;
selection-background-color: lightgray;
}
QComboBox QAbstractItemView::item {
height:2em;
}</string>
        </property>
        <item>
         <property name="text">
          <string>c++</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>python</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>java</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>kotlin</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 20px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>微软雅黑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 33px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>微软雅黑</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FramelessWindows.qrc"/>
 </resources>
 <connections/>
</ui>
